byron/ledger/executable-spec/cs-ledger.cabal

```diff
@@ -109,3 +109,36 @@ test-suite ledger-delegation-test
                "-with-rtsopts=-K1m -M50m"
   if (!flag(development))
     ghc-options: -Werror
+
+test-suite ledger-sts-test
```

Comment:
There is already a `test-suite ledger-delegation-test` section. Would it make sense to move the contents of the that section to this one (since the tests there are also testing the STS).

Reply:
Sure, that's completely doable, though the test suite's name would be misleading in such a case.

Reply:
I merged the two test suites into one.